sklearn_hierarchical_classification/classifier.py

```diff
@@ -527,7 +544,12 @@ def _base_estimator_for(self, node_id):
         return clone(base_estimator)
 
     def _make_base_estimator(self, node_id):
-        return LogisticRegression()
+        """Create a default base estimator if a more specific one was not chosen by user."""
+        return LogisticRegression(
+            solver="lbfgs",
+            max_iter=1000,
+            multi_class="multinomial",
+        )
```

Comment:
unrelated, this gets rid of the ConvergenceWarning I started seeing in unit-tests